sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/AccountDetails.kt

```diff
@@ -0,0 +1,11 @@
+package cash.z.ecc.android.sdk.internal.model
+
+internal data class AccountDetails(
+    val account: Int,
+    val extendedFullViewingKey: String,
+    val address: String,
+    val transparentAddress: String
```

Comment:
The `accounts` table no longer has these columns, and instead has a `ufvk: String` column. There is a separate `addresses` table that tracks the addresses that have been derived for a given account.